AntennaTracker/AntennaTracker.cpp

```diff
@@ -50,7 +50,8 @@ const AP_Scheduler::Task Tracker::scheduler_tasks[] = {
     SCHED_TASK_CLASS(AP_Notify,         &tracker.notify,    update,         50,  100),
     SCHED_TASK(one_second_loop,         1,   3900),
     SCHED_TASK(compass_cal_update,     50,    100),
-    SCHED_TASK(accel_cal_update,       10,    100)
+    SCHED_TASK(accel_cal_update,       10,    100),
+    SCHED_TASK(update_adsb,            10,    100),
```

Comment:
rework to be:
`SCHED_TASK_CLASS(AP_ADSB,           &tracker.adsb, update,     10, 100),`